psgallery_status.md

```diff
@@ -3,7 +3,13 @@ PowerShell Gallery Status
 =========================
 ### The [PowerShell Gallery](https://powershellgallery.com) is currently:  __Operating with Statistic Errors__
 
-### 10/15/2020 - PowerShellGallery.com may temporarily experience latency and/or low availability 
+30/10/2020 - PowerShellGallery.com has CDN is down preventing packages from being downloaded and availability for the gallery itself has been extremely low 
```

Comment:
`PowerShellGallery.com has CDN id down` should read `PowerShellGallery.com CDN is down`